lib/api/CMakeLists.txt

```diff
@@ -91,16 +91,19 @@ if(WASMEDGE_BUILD_SHARED_LIB)
 
 endif()
 
-if(WASMEDGE_BUILD_STATIC_LIB)
+if(WASMEDGE_BUILD_STATIC_LIB) 
 
+  # FIXME: spdlog::spdlog will fail on Darwin.
```

Comment:
Don't we have CI for building and testing this?

Reply:
Our CI doesn't test for `-DWASMEDGE_BUILD_STATIC_LIB`.
Only the dynamic library are tested, because the static library is not often to be used.

Reply:
For the current status, the `proxy-wasm` uses our static linking.